ingq/illust/utils/mapper.py

```
@@ -45,3 +49,17 @@ def illustvo_to_illust_item(illust: IllustVO) -> IllustItem:
             created_at=illust.created_at,
             updated_at=illust.updated_at,
         )
+
+    @staticmethod
+    def illustsvo_to_illust_item_list(illusts: list[IllustVO]) -> IllustItemList:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)